### DISCUSSION TITLE
[Bug] 个人主页动态显示为“出现未知错误，已记录”

### DISCUSSION THREAD

**Discussion by SuanOrange** [ywmoyue/biliuwp-lite#556] [bug]
### Bug 描述

扫码登陆后动态页无内容，显示“出现未知错误，已记录”，无法显示正常内容

### 操作步骤

```plain text
1.打开应用(版本为v4.6.28.0)
2.点击右上角头像，显示“个人中心”
3.点击“个人中心”，显示“动态”按钮
4.点击“动态”，显示“出现未知错误，已记录”
```

### 期望结果

正常显示个人空间动态

### 截图

![Screenshot 2024-03-31 103414](https://github.com/ywmoyue/biliuwp-lite/assets/67503405/d8d25a4f-4d0c-484e-ae18-4f85a0a88a36)

### 应用来源

Github/其他Github镜像站

### 安装方式

脚本安装

### 系统版本

其它 (需注明)

### 其它系统版本

Windows 10 22H2 (Build 19045

### 设备类型

笔记本

### 设备架构

x64

### 应用架构

x64

### 备注

其他用户主页动态依旧如此

### 代码贡献

我更擅长发现问题

**Comment by ywmoyue** [comment]
感谢反馈，这个正在处理中
